src/Cli.Tests/ValidateConfigTests.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Cli.Tests;
+/// <summary>
+/// Test for config file initialization.
+/// </summary>
+[TestClass]
+public class ValidateConfigTests
+    : VerifyBase
+{
+    private IFileSystem? _fileSystem;
```

Comment:
how come we need this `IFileSystem` as its own field? can't this just live in the loader?

Reply:
we are using that fileSystem to create the loader. following the same convention as other tests.